02-peer-protocol.md

```diff
@@ -2773,18 +3325,33 @@ A node:
 
 A receiving node:
   - if `next_funding_txid` is set:
-    - if `next_funding_txid` matches the latest interactive funding transaction:
-      - if it has not received `tx_signatures` for that funding transaction:
```

Comment:
I may have missed this, but what is the rationale for removing the
```- if it has not received `tx_signatures` for that funding transaction:```
condition for the receiving node?

#1214 still keeps that condition.